src/Ursa.ReactiveUI/Ursa.ReactiveUIExtension.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0;net8</TargetFrameworks>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<LangVersion>latest</LangVersion>
+		<Authors>WCKFWCKY</Authors>
```

Comment:
If we can co-author like WCKFWCKY, IRIHI Technology? ( you can ignore if you don't want to)